docs/development/release-checklist.md

```diff
@@ -20,22 +18,21 @@ Pre-Release Tasks
    - Update supported NetBox versions in `tests/integration/conftest.py` if needed
    - Ensure the `get_netbox_docker_version_tag` function in `tests/integration/conftest.py` is updated with any new version mappings
 
-Release Tasks
------------
+## Release Tasks
 
 1. Create a new release branch from `master`:
 
-   .. code-block:: bash
-
-       git checkout master
-       git pull
-       git checkout -b release/vX.Y.Z
+   ```bash
+   git checkout master
+   git pull
+   git checkout -b release/vX.Y.Z
+   ```
 
 2. Commit version and changelog updates:
 
-   .. code-block:: bash
-
-       git commit -m "Prepare release vX.Y.Z"
+   ```bash
+   git commit -m "Prepare release vX.Y.Z"
+   ```
 
 3. Create a pull request to merge the release branch into `master`
 4. Once merged, use github to create a new release:
```

Comment:
These lists aren't rendering quite right. For instance, this second list has items: 1, 1, 2, 3, 4, 5, 6, 7, and 8 instead of 1-9. I think indenting the code blocks might help with that.

Reply:
This list is looking good now, but the one above is still borked. Apologies, I should have been more clear before.